### PR TITLE
[UI/seller/editprofile] - create seller/editprofile page and change seller/my-page

### PR DESCRIPTION
**Pull request by Sayuriishida**
Additions:
- add sellsers/modal/editprofile.blade.php

Changes:
- add "Company Name" display area in sellers/index.blade.php
- change layout in sellers/index.blade.php

### CODE REVIEW

File: resources/views/sellers/index.blade.php [hunk 1]
```diff
@@ -2,87 +2,100 @@
...
 
-        <div class="row">
-            <div class="col-12">
-                <p class="mb-0">Street Address</p>
-                <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
-            </div>
+            <div class="row">
+                <div class="col-12">
+                    <p class="mb-0">Street Address</p>
+                    <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
```
Comment (Ninit29) [inline resources/views/sellers/index.blade.php:52]: I suggest to put the style on a css file and assign a class

Comment (Ninit29) [reply resources/views/sellers/index.blade.php:52]: especially when you're repeatedly using the same style
background-color: #E6E6E6;

File: resources/views/sellers/index.blade.php [hunk 2]
```diff
@@ -2,87 +2,101 @@
...
 
-        <div class="row">
-            <div class="col-12">
-                <p class="mb-0">Street Address</p>
-                <p class="rounded py-2 px-3" style="background-color: #E6E6E6;">4140 Parl</p>
-            </div>
+            <div class="row">
+                <div class="col-12">
+                    <p class="mb-0">Street Address</p>
+                    <input class="rounded py-2 px-3" value="4140 Parl" readonly>
```
Comment (Ninit29) [inline resources/views/sellers/index.blade.php:53]: add type attribute
<input type="text"

check also other inputs

File: resources/views/sellers/index.blade.php [hunk 3]
```diff
@@ -2,87 +2,101 @@
...
+                    <input class="rounded col-12 py-2 px-3" value="20033">
+                </div>
 
-            <div class="col-6">
-                <p class="mb-0">password</p>
-                <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">**********</p>
-            </div>
+                <div class="col-6">
+                    <p class="mb-0">password</p>
+                    <input tyle="password" class="rounded col-12 py-2 px-3" value="**********">
```
Comment (Ninit29) [inline resources/views/sellers/index.blade.php:71]: typo `input `instead of `tyle`

File: resources/views/sellers/index.blade.php [hunk 4]
```diff
@@ -2,87 +2,101 @@
...
+                </div>
 
-            <div class="col-6">
-                <p class="mb-0">Credit Card Number</p>
-                <p class="rounded col-12 py-2 px-3" style="background-color: #E6E6E6;">***************</p>
-            </div>
+                <div class="col-6">
+                    <p class="mb-0">Credit Card Number</p>
+                    <input tyle="password" class="rounded col-12 py-2 px-3" value="***************">
```
Comment (Ninit29) [inline resources/views/sellers/index.blade.php:76]: typo `type `instead of `tyle`

Comment (Sayuriishida) [reply resources/views/sellers/index.blade.php]: I fixed the word with [da95828](https://github.com/kredo-school/16th_rankshty/pull/49/commits/da958284dccea7624d70416e87d40bbca078e805) and [7540421](https://github.com/kredo-school/16th_rankshty/pull/49/commits/754042125237aab492ea60a3928db9d581a2c4c8).

File: resources/views/sellers/index.blade.php (hunk 3 above)
Comment (Sayuriishida) [reply resources/views/sellers/index.blade.php]: I fixed the word with [da95828](https://github.com/kredo-school/16th_rankshty/pull/49/commits/da958284dccea7624d70416e87d40bbca078e805) and [7540421](https://github.com/kredo-school/16th_rankshty/pull/49/commits/754042125237aab492ea60a3928db9d581a2c4c8).

File: resources/views/sellers/index.blade.php (hunk 2 above)
Comment (Sayuriishida) [reply resources/views/sellers/index.blade.php]: I fixed the word with [da95828](https://github.com/kredo-school/16th_rankshty/pull/49/commits/da958284dccea7624d70416e87d40bbca078e805) and [7540421](https://github.com/kredo-school/16th_rankshty/pull/49/commits/754042125237aab492ea60a3928db9d581a2c4c8).

**Review by Ninit29 — APPROVED** [review]

**Pull request merged**